.tekton/fbc-v4-16-push.yaml

```diff
@@ -0,0 +1,358 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/parodos-dev/orchestrator-fbc?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "push" && target_branch
```

Comment:
you're missing the cel expressions to filter out the pipeline to only happen when there are changes to the v4.16 directory

Reply:
Do we want this for the push event? I thought it was just for the PR events as with the 4.15, 4.14

Reply:
Updated cel expression for all push pipelines.